### ISSUE TITLE
DRAFT SC Minutes - Temporary for creating markdown

### ISSUE THREAD

**Issue by jude2018** [OpenWIS/openwis-documentation#471]
**OpenWIS Association Steering Committee Meeting 2018 May**

30 to 31 May 2018

1. Welcome and introductions
Jeremy Tandy (SC chair) welcomed all present and Weiqing Qu who joined remotely.

2. Approval of agenda
All attendees were content with the agenda.

3. Declaration of delegates
    1.The current members of the SC are:
         Jeremy Tandy (JT), Met Office (UKMO) - SC Chair
         Rémy Giraud (RGr), Météo France (MF) - SC Vice-Chair
         Loïc Le Gallou (LG), Météo France International (MFI)
         Weiqing Qu (WQ), Bureau of Meteorology (BoM)
         Sungsoo Do (SD), Korea Meteorological Administration (KMA)
         Kari Sheets (KS), National Weather Service (NWS)
         Mikko Visa, Finnish Meteorological Institute (FMI)

  All attendees were content with the following:
  Steve Olson (SO) representing National Weather Service (NWS) with authority from Kari Sheets

  Rémy Gibault (RGb) representing Météo France International (MFI) with authority from Loïc Le 
  Gallou.

  Rémy Giraud (RGr) from Météo France MF was elected as the vice chair.  Seconded by Jeremy 
  Tandy (JT) and agreed unanimously.

   Other attendees are:
   Met Office (UKMO) – Jeremy Tandy (JT) (SC Chair)
   
   European Centre for Medium Range Weather Forecasts (ECMWF) – Stephan Siemen (SS).  It was 
   noted that ECMWF signed up as Associate Partner, so have right to attend and participate but no 
   vote.  All welcomed ECMWF.
   
  Stacey Harvey (SH) (Treasurer)

RESOLUTION - Rémy Giraud (RGr) from Météo France MF was elected as the vice chair.  Seconded by Jeremy Tandy (JT) and agreed unanimously (5 votes).

SC-2018:#4 Approval of previous minutes
The previous meetings are here:
a.	22 Mar 2017: http://openwis.github.io/openwis-documentation/minutes/2017-03-22-OpenWIS-Steering-Committee.html
b.	07 Dec 2017: http://openwis.github.io/openwis-documentation/minutes/2017-12-07-OpenWIS-Steering-Committee-2017-Dec.html

•	There were no queries or comments on the previous minutes.  
•	The chair and vice-chair thanked Paul Rogers for bringing these minutes together.

SC-2018:#5 Election of chairperson ("TC-Chair") and vice-chairperson ("TC-Vice-Chair") of the Technical Committee

•	Steve Olson and Benjamin Saclier (BS) agreed to stand as Chair and Vice Chair respectively.  
•	There were no other proposals for these roles.  
•	There was unanimous acceptance that Steve and Benjamin could continue in these roles. 

RESOLUTION – There was unanimous acceptance (5 votes) that Steve Olson and Benjamin Saclier would continue in the roles of Chair of the Technical Committee and Vice Chair of the Technical Committee, respectively.

SC-2018:#6 Recommendations to the Board regarding admission of new Members and Partners (see Articles 6 & 9 and Rules 4 & 5)

a.	Progress update: ECWMF (Associate Partner) [Issue #347] 
Rémy Giraud (RGr) stated that the ECMWF had signed-up late last year (2017).  

RGr - The challenge was on VAT last year.  It need to be clear what each partner pays and what the VAT element should be.  Got agreement from Belgian lawyer that no VAT needs to be paid.  Therefore, it is to be noted that official confirmation was received from the Belgian Tax Authority that the OpenWIS Association is not subject to VAT.  
 
It was agreed that once the DropBox for Business service is in place for secure storage of Contributor Licences, Rémy Giraud to place the VAT related documentation in this DropBox repository. 

[Action-SC-2018-18 place VAT documents in DropBox for Business]( https://github.com/OpenWIS/openwis-documentation/issues/424)

All attendees welcomed ECMF and it was agreed that Stephan Siemen and Baudouin Raoult should be added to the mailing list for the OpenWIS SC and TC. 

[Action-SC-2018-19 Add ECMWF representatives to mailing list for SC and TC](https://github.com/OpenWIS/openwis-documentation/issues/425)

b.	Progress update: IMD (Strategic Partner) [RESOLUTION-BM-2017-12]

By a previous resolution of the Board it was agreed that Indian Met Dept. (IMD) is to be part of the OpenWIS Association as an Associate Member.  

It was agreed that further discussions regarding IMD should be postponed until Matteo Dell’Acqua (MDA) is available at the Board meeting.

c.	Other organisations: 
i.	CMA 
ii.	NMI [Issue #325] 

It was agreed that discussion on this item (i.e. re CMA and Norway) should continue at the Board meeting when MDA is available. 

Other points raised by SO:

ECMWF needs to be added this OpenWIS Association web site page - 
      http://openwis.github.io/openwis-documentation/structure/members.html

[Action-SC-2018-20 add ECMWF to the website](https://github.com/OpenWIS/openwis-
      documentation/issues/426)

The logos for each participating organisation appear as different sizes on the OpenWIS 
      Association web site, i.e. on this page -   
      http://openwis.github.io/openwisdocumentation/structure/members.html 

[Action-SC-2018-21 resolve the different sizing of logos on the web 
     site](https://github.com/OpenWIS/openwis-documentation/issues/427)

The page title of the membership page of the website (i.e. this page http://openwis.github.io/openwis-documentation/structure/members.html) needs to be altered: the Page Title ‘OpenWIS Members and Partners’; the Sub Heading is ‘members’ instead of ‘core founders’; and a new sub-heading added to say  ‘Associate partners’.  

[Action-SC-2018-22 change the membership web 
      page](https://github.com/OpenWIS/openwisdocumentation/issues/428)

SC-2018:#7 Recap on governance and organisation roles within OpenWIS Association

A recap of issue #185 and MS PowerPoint slide (i.e. OpenWIS) took place and the following was agreed by the SC:

Amend the wording for membership of PMC; remove word “initially” and append a statement “membership may evolve over time”; also change “participating” to “sponsoring” is needed.

[Action-SC-2018-23 update the “responsibilities” rule supplement and the “Project Initiation” PowerPoint slide](https://github.com/OpenWIS/openwis-documentation/issues/430)

The approval of [software] releases by a Project is optional - it will depend on the release regime agreed with the SC during the initiation of the Project; e.g. self-assessment by PMC.

“Project Operation” slide currently only describes software releases; OpenWIS Projects may be “study” activities - use “project deliverables” in place of “software package(s)” and amend TC responsibility regarding “validation of technical design and implementation” etc.

[Action-SC-2018-24 update the “responsibilities” rule supplement and the “Project Operation” PowerPoint slide]( https://github.com/OpenWIS/openwis-documentation/issues/431)

Amend “Project Operation” to indicate that PMC (or TC) may recommend follow-on actions or projects for consideration by the SC.

[Action-SC-2018-25 update “responsibilities” rule supplement](https://github.com/OpenWIS/openwis-documentation/issues/432)

There is a need to clarify that approvals by the SC and Board are recorded in minutes that are published for public viewing; these will specifically list RECOMMENDATIONS and RESOLUTIONS made.

RESOLUTION - Ensure that approvals by the SC and Board are recorded in minutes that are  published for public viewing to specifically list RECOMMENDATIONS and RESOLUTIONS made.

[Action-SC-2018-26 update “responsibilities” rule supplement]( https://github.com/OpenWIS/openwis-documentation/issues/433)

Amend the Allocation of funds slide, change the SC approval to "in budget and <EUR10000". 

[Action-SC-2018-27 update the “allocation of funds” PowerPoint slide]( https://github.com/OpenWIS/openwis-documentation/issues/434)

SC-2018:#8 Project Reports and requests for Financial Support

a.	OpenWIS Core
The report from the PMC Lead (SO): 

This project is in maintenance mode.  It is the largest delivery in the OpenWIS Association that 
centres on security. Releases will be is in 3 aspects: Java, Spring and JBOSS container.  
Project deliverable was March 2018 with artefacts and is in the CI process today.  The plan is to 
have the release available before the European Dynamics contact (i.e. within the next few   
weeks).  At this time, no finance is required from OPENWIS ASSOCIATION.

RESOLUTION - The SC is content for the OpenWIS Core PMC to proceed with releases based on 
     self-assessment. 

b.	Excel2Wis
The SC accepted the report from the Project Lead (BS).

c.	OpenWIS Core Harness [empty]
MF are planning to invest further in the harness in the coming year are likely to provide a new Project Charter on the dissemination harness in the coming months.

[Action-SC-2018-28 remove the current repository within Github](https://github.com/OpenWIS/openwis-documentation/issues/435)

d.	WIS2 Proof of Concept.
The SC are content that the project delivered against the Project Charter, that it should be closed, and note that a number of follow-on studies are being proposed.  

[Action-SC-2018-29 amend the OpenWIS association website](https://github.com/OpenWIS/openwis-documentation/issues/436)

RESOLUTION - The SC is content that the WIS2 Proof of Concept (PoC) can be closed.

SC-2018:#9 Project Report and recommendations from Technical Committee

The Report from TC Chair (SO):
Technical presentations were made regarding the current technical infrastructure and CI process. The TC recommends the continued use of the existing infrastructure and CI process. The TC notes that there are current deficiencies with test coverage. The TC recommendation is to address test coverages with next generation of OpenWIS (i.e. OpenWIS2) software due to OpenWIS 3 being in maintenance mode.
For Code of conduct and Contributor License Agreement (CLA), the TC recommends internal founding/strategic member organizations who are contributing code, sign CLA agreements by July 2018.  Furthermore, the TC recommends to SC the provisioning of secure storage for CLA signed artefacts. The TC also passed a resolution on updates to Code of conduct regarding procedural changes for non-compliance.
For Technical rules, the TC took an action to realign/restructure project wiki content and website to streamline access to pertinent information. Several outstanding issues were closed, and others will be closed after this restructuring.
For Project design authority, standards compliance, and release management, the TC took an action to provide guidance w.r.t. standards compliance as a supplement to Technical rules. The release management process is captured in the restructuring efforts listed immediately above.
For Key risks and issues of individual OpenWIS Projects, the TC recommends continued maintenance of OpenWIS 3 for the next 5-6 years. As part of the maintenance plan, the TC created a risk mitigation plan to address key influences to OpenWIS software stack. For the excel2WIS project, no risks have been identified. The TC view this project as being "dormant" now. Regarding quality measures, the TC will establish a webpage within technical governance that describes repeatable processes/ and wherever possible highlight quality reporting metrics.
RESOLUTION – The SC approved the TC recommendation for continued maintenance of OpenWIS 3 for the next 5-6 years.
Future work. The TC recommends to SC approve 4 new projects to explore/investigate WIS2 specific requirements. The TC made a resolution to establish and maintain a cloud VM environment for collaborate development activities for OpenWIS association as required. The TC will provide the technical specification and associated costs needed to support this environment. The TC seeks SC approval of this resolution and solution. Furthermore, the TC recommends to SC that a recurring budget item be established to cover the costs of this cloud environment.
RESOLUTION – The SC approved the TC recommendation for the 4 new projects to explore/investigate WIS2 requirements.
Forum: The TC is in the process of establishing 1-year license agreement of discourse. The TC formalized the initial set of categories, subcategories, topics, etc. to be used. The TC recommends the use of openwis.discourse.org as domain name for forum. The TC recommends to SC that a recurring budget item be established to cover the costs of discourse license.
RESOLUTION – The SC approved the TC recommendation for use of discourse and need for a recurring budget to cover licence costs.
The SC accepted all recommendations and TC resolutions made by the TC Chair.
SC-2018:#10 OpenWIS Association logistics: programme management / secretariat function

With Paul Rogers retiring, it seems pertinent to review the hosting of the Secretariat function.

The SC formally thanked Paul Rogers for enabling the OPENWIS ASSOCIATION run smoothly.

SC voted unanimously for the Met Office to continue with provision of Secretariat function
RESOLUTION –The SC unanimously voted (5 votes) in favour of the Met Office continuing the Secretariat Function (Met Office representative abstained from voting) 
JT - Met Office content to continue - subject to review once again at Annual Meeting, 2020 (2-years time, i.e. not an open ended offer)

RGr - it may be appropriate for Met Office to charge back to the Association for the staff costs required to run the Secretariat (as distinct from Project activities).

JT  - the estimated effort is 15% FTE

Stacey Harvey was asked to provide external ‘at cost’ charge rate (in €, no profit, loaded costs) for 15% of a Senior Project Manager.

Information provided by SH: the price at cost plus 4.35% for overheads (e.g. laptop/office to work in etc.) then looking at €11,308 for 0.15FTE or a day rate of €290. These costs are not specific and are based on a Senior Project Manager in Technology.

SC-2018:#11a OpenWIS Shaping the OpenWIS Association
Status of WIS 2.0 (report from CBS TECO 2018)
i.	WIS 2.0 implementation approach
ii.	Outcomes from OpenWIS Association demonstration (PoC): Data publication to WIS 2.0 and real-time distribution

These items were discussed as part of the TC items in the PMC annual meeting.
 
SC-2018:#11b How/where can/should OpenWIS Association add value?
i.	Supporting WIS 2.0 implementation 

The SC have identified four study projects that will help with the design of WIS2 as  recommended by the TC

ii.	As an umbrella for meteorological open source software projects - including mature projects developed elsewhere that would benefit from a home (e.g. Smartmet, MetPX)

   MV – there are lots of overlap between WIS2.0 discussions and those in the INSPIRE 
Maintenance & Implementation Group (MIG); e.g., use of DCAT, RESTful Web services, 
protocols etc., and as FMI is already involved in these activities; would be possible to share 
insights gained here.

   The SC agreed that FMI should lead on curating recommendations from INSPIRE and OGC that are relevant for WIS2 in a series of short reports.  This is work that FMI is already doing and demonstrates the values of collaboration as it saves duplication of effort in each organisation. 

   [Action-SC-2018-30 create project charter](https://github.com/OpenWIS/openwis-documentation/issues/438)

JT - OpenWIS Association is providing value as a forum for pooling our resources and sharing insight 

SS - OpenWIS Association is a place where we can cooperate on [technical] solutions

JT - the power of this collaboration is amplified when we are able to share the results of work that we have already committed to undertake - [unanticipated] re-use

SC agreed that the main purpose of OpenWIS Association, for this coming year, is to pool our resources to further the understanding of the problems and challenges facing the broader meteorological community so that we can inform and influence the design of WIS2.

iii.	Application of Association cash assets.
RESOLUTION – The SC agreed that the best way to fund work for this year is to begin draw down of cash assets. 

SC-2018:#11c Review the continuing need for the OpenWIS Association
The SC agreed to pool resources so that we can better influence the design of WIS2.

SC-2018:#11d Review proposals for projects that are compatible with the purpose and objectives of the OpenWIS Association and in the interest of Members and Partners
i.	FMI SmartMet [Issue #186] 

The SC agreed that this is time-expired and that the project can be closed.

ii.	OpenCDMS [no change since SC 2017] 

The SC agreed that there are no resources therefore not in a positon to proceed as a project.

iii.	Weather API Specifications [Issue #188, Issue #349] [project approved: RESOLUTION-SC-2017-19, Met Office lead: RESOLUTION-SC-2017-18] Delay in development of API specification in the UK Met Office. 

The Weather API will be specified in the MetOcean DWG.  Therefore, the SC decided to suspend this project until the OpenWIS Association knows the outcomes of this.

iv.	How to use Big Data (demonstrators) [Met Office proposal] 

JT – presented a proposal for prototype software developed within the Met Office that has the potential for others begin leverage of the extensive global NWP and satellite data.

RGr - no objection to this in principle, but concerned about alignment of these prototypes with the vision and technical direction of OpenWIS Association.  

RGr - establishing a Project implies endorsement and could dilute the OpenWIS brand.

JT - how would we be sure that the prototypes were consistent with the vision and technical direction that the Association wants to promote?

RGr - we develop PoC software for our own needs internally - OpenWIS Association would want to look inside the box to see if it is aligned with what OpenWIS Association wants to promote; e.g., it is demonstrating an appropriate technical direction / architecture approach.  

RGr - as a general principle, would like to look at the Project output before endorsing it.

JT- if you want to vet each prototype, this will add friction to the internal [Met Office] development teams - the added governance would be prohibitive.

The SC agreed that this proposal should be withdrawn in its current form and the way forward is for RGr and JT to discuss further how Projects like this proposal might operate; noting the need to balance flexibility for internal development with the need for OpenWIS Association validation

[Action-SC-2018-1 Propose how to validate project proposals] 
(https://github.com/OpenWIS/openwis-documentation/issues/401)

v.	Further WIS2 (PoC) projects? 

The SC agreed that there were none at present.  However, it is noted that actions are present for Project Leads to develop Project Charters for Board approval in the coming weeks by end of June 2018. 

[Action-SC-2018-31 - create project charter for schema.org 
       study](https://github.com/OpenWIS/openwis-documentation/issues/439)

[Action-SC-2018-30 create project charter]
( https://github.com/OpenWIS/openwis-documentation/issues/438)

[Action-SC-2018-32 create project charter for AMPQ]
(https://github.com/OpenWIS/openwis-documentation/issues/440)

[Action-SC-2018-33 create project charter for trust model]
(https://github.com/OpenWIS/openwis-documentation/issues/441)

SC-2018:#11e Review requests for amendments to Charters of existing Projects
There were no requests to amendments.

SC-2018:#11f Recommendations to the Board regarding new or amended Projects 

None at present. However, actions are present for Project Leads to develop Project Charters for Board approval in the coming weeks by end of June 2018 (see SC-2018:#11d.v)

SC-2018:#11g Identification of opportunities for engagement with WMO
i.	WIS 2.0 implementation.  
      The SC have discussed this extensively in previous agenda items.

ii.	Executive Council 2018 [Issue #205]
The SC requests that its delegates work with their Permanent Representatives (PRs) to support this Draft Decision EC-70/7.2(1)/2 that is referenced here - https://goo.gl/2B5BVA 

RGr - the side-meeting planned for the Cache in the Cloud at EC-70, Friday 22 June.  PRs of GISCs are invited to commit to participate in the Cache in the Cloud project and to determine the appropriate funding mechanism.  

RGr - No formal funding commitment at EC-70; this will come via a formal letter from WMO Secretariat following agreement. 

RGr- at CBS-TECO the difficulties arose around determining the funding mechanism and whether or not it was affordable based on an equal share (e.g. CHF20k / year for 3-years).

RGr - Draft Decision EC-70/7.2(1)/4 describes the proposal further and identifies use of a Trust Fund with contributions from Members operating GISCs for the funding mechanism; see https://goo.gl/2B5BVA

The SC agreed that all SC delegate should work with their Permanent Representatives (PRs) to gain commitment for Cache in the Cloud during this side event. Cache in the Cloud project is a precursor to WIS 2.0 - particularly in determining how we can deliver shared services.

[Action-SC-2018-2 support cache in the cloud](https://github.com/OpenWIS/openwis-documentation/issues/406)

SC-2018:#11h Identification of opportunities to participate in calls for proposals in externally funded projects 
i.	EU: Horizon 2020, Copernicus
ii.	US 
iii.	Australia

No opportunities were identified.  The SC agreed that, given the sensitivities around resourcing, 
       any proposal around a project needs to be thoroughly thought through.

      The SC agreed that SS should monitor Horizon 2020, Copernicus proposals and others relevant to 
       the OpenWIS Association and to alert the SC of potential options.

[Action-SC-2018-3 funding proposals]( https://github.com/OpenWIS/openwis-documentation/issues/407)

      
SC-2018:#11i Recommendations to the Board regarding engagement with third-parties to participate in consortia responding to calls for proposals in respect of externally funded projects.  

The SC’s recommendation to the Board is that there are no externally funded project proposals.

SC-2018:#12 Review of Strategic Goals and Metrics (3-year horizon) 
[metrics] [Issue #215, Issue #216]

The SC reviewed the metrics and issues and agreed the following:

There may be other metrics that are relevant to projects, such as one relating to having multiple 
      projects. Therefore, the SC agreed that metric 
      A1.2 should be reviewed.
[Action-SC-2018-4 Review and update metric A1.2](https://github.com/OpenWIS/openwis-documentation/issues/408)

The SC agreed the removal of metric A1.1.  

[Action-SC-2018-5 Remove goal A1.1 from website]( https://github.com/OpenWIS/openwis-documentation/issues/409)

    The SC suggested the following new goals:

•	OpenWIS Association to influence and inform the design of WIS2.   A metric is required.
•	Increase the number of OpenWIS Association partners. A proposed metric for this goal - have a clear value offering for the WMO community.
•	Encourage active contribution from all Members and Partners.  Suggested metrics and mechanisms: 
o	Metric moving toward more equitable contributions; balanced in-kind contributions.
o	Metric: see the accounts 
o	Metric: technical contributions on GitHub
o	Mechanism: to share insight within OpenWIS Assoc Members and Partners gained by each organisation from their own works (e.g. in OGC, WMO Expert Teams, INSPIRE MIG etc.)

[Action-SC-2018-34 review and update goals and metrics](https://github.com/OpenWIS/openwis-documentation/issues/442)

SC-2018:#13 Risk review and development of mitigation plans [Issue #170, Issue #191, Issue #227, Issue #351] 

RESOLUTION -The SC approved the risk management process proposed by Paul Rogers in issue #227. 

RESOLUTION - The SC agreed to review entire risk log twice yearly.  If a risk (existing or new new) requires urgent attention this would be done ex-committee.

[Action-SC-2018-6 Create a risk management rules supplement](https://github.com/OpenWIS/openwis-documentation/issues/410)

Review of specific existing and new risks:
a.	Existing risks:
i.	[GREEN] RISK-000001 Insufficient funds. 
•	The SC agreed the assessment of this risk should remain unchanged. 

ii.	[RED] RISK-000002 Compliance failures.
The mitigations agreed by the SC is to continue: with the current Tax Advisor; and to prepare a register of obligations to the Belgian Authorities. 

The SC agreed that a register of obligations to the Belgian Authorities is needed so that all actions necessary are known in advance. 
[Action-SC-2018-7 set up a register of obligations to the Belgian Authorities]( https://github.com/OpenWIS/openwis-documentation/issues/411)

iii.	[RED] RISK-000003 Under resourced.  
The SC agreed these mitigations and revised risk assessment:
•	Employ the cash reserve of OpenWIS Association for study work contracted out to third parties.
•	Work to get more active contribution from each member and partner organisations. TC and SC delegates to work with their home organisation to better encourage in-kind contributions.  
•	Creation of a new OpenWIS 3 project to help focus activity in this regard.  

With these mitigations in place, change overall risk assessment rating to Amber, noting that the approach of using cash-reserves is not sustainable long-term.

b.	Proposed risks: 
i.	Fragmentation of OpenWIS Community
   The SC assessment is GREEN.  Mitigations are the new study projects and the focus to 
               support design of WIS 2

ii.	Disruption from on-boarding new Partners.
The SC assessment is GREEN.  Mitigations include restructuring and improving 
               documentation.

iii.	Loss of OpenWIS visibility in WIS 2.0
The SC agreed that this is the risk that the OpenWIS Association is too strongly related to the 
              original OpenWIS Core software making it difficult to get new Partners or influence the 
              design on WIS 2.0.

The SC assessment is AMBER. Mitigations include: outreach and 
communications that clarify the goals of the Association regarding the support of transition to WIS2; review the Association's Projects to ensure that they are geared towards supporting WIS2 rather than focusing exclusively on OpenWIS Core

iv.	Exposure to Intellectual Property claims [Issue #190, Issue #342]
The SC assessment is GREEN.  The risk comes from incomplete coverage of Contributor 
License Agreements (CLAs)  (e.g. all Contributors have not signed).  Mitigations include: getting the CLAs signed (orchestrated by the TC) and getting them stored securely on Dropbox for Business.  See Action-SC-2017-19 Submit completed CLA forms 
 https://github.com/OpenWIS/openwis-documentation/issues/190 and Action-SC-2017-72 
 Finalize the CLA as-is and obtain signed copies from all contributors 
 https://github.com/OpenWIS/openwis-documentation/issues/342

v.	Misalignment of Technical Committee definition [Article 14.1] and amended Association governance.

The SC assessment is GREEN and to accept the risk.   The definition of the Technical Committee, as presented in Article 14.1 http://openwis.github.io/openwis-documentation/articles/14-technical-committee.html#article-14.1, does not reflect the amended governance arrangements required to support multiple projects; the TC takes an overarching role (e.g. design authority?) while PMCs undertake tasks 1-4 and 6 defined in Article 14.1

vi.	Major projects with significant community visibility fail to deliver.
The SC assessment is GREEN.  The reputation impact would be high and the probability very low. No mitigations required at present.

vii.	OpenWIS Association fails to remain relevant to WMO.
The SC assessment is GREEN.  The mitigations include include: influencing the evolution 
               and design of WIS 2.0; this is achieved by contributing to the work of WMO Expert and 
               Task Teams

[Action-SC-2018-8 update the risk log]( https://github.com/OpenWIS/openwis-documentation/issues/412)

SC-2018:#14 Finances of the OpenWIS Association [Treasurer]

a.	FY2017 expenditure
The Treasurer’s report was reviewed as the following agreed by the SC:

•	ECMWF membership fee - It is necessary to accrue ECMWF Associate Partner contribution from 2017 as ECMWF’s membership of OpenWIS Association in 2017. 

RESOLUTION – The SC agreed that the funds could paid into the ING bank for this transaction.

[Action-SC-2018-9 raise an invoice of 10,000 Euros for ECMWF Associate Partner fees](https://github.com/OpenWIS/openwis-documentation/issues/413)

[Action-SC-2018-10 clarify 'in kind' position of UKMO re Amazon Web Services (AWS)](  https://github.com/OpenWIS/openwis-documentation/issues/414)

OVH Servers - This will be treated in 2017 as in-kind contributions by MF.  Note that this 
         expenditure will not be required in 2018.

Domain name – This is an annual payment should continue to be budgeted for in 2018. 

Hosting Discourse servers - NWS will treat these costs as ‘in kind’ for 2017 and will invoice be 
         for in 2018.  This invoice will be in July 2018.  Note that half of the costs will be in 2018 and the  
         remainder in 2019.

Promotional costs - these costs will not need to be budgeted for in 2018.

Legal costs - Costs incurred in 2017 related to expenditure for 2016 and 2017.  Only €80 was 
         accrued for 2016 accounts filing.  However, cost came to €1,096.  A further €1,096 was also 
         accrued for 2017 costs (€1096 x 2 = €2192 -€80 =€2,112). 

Services – Consultancy Support (Tax Expert) – Lawyer services in connection with VAT 
         investigation.

Late fees - Clarification required on whether this needs to be paid in 2018. 

[Action-SC-2018-11 check costs due from historic work to prepare annual   
         accounts](https://github.com/OpenWIS/openwis-documentation/issues/416)

Bank fees - the SC agreed the general contingency of 2000 Euros would cover any bank charges from MFI or ING.  

The bank charges due from MFI needs clarification.  The SC agreed that RGb and SH work together to understand the MFI bank charges.

[Action-SC-2018-12 clarity MFI bank charges]( https://github.com/OpenWIS/openwis-documentation/issues/417)

OpenWIS Association Secretariat service – This is provided by the Met Office and requires a budget in 2018 of 11,500 Euros (based on 15% FTE of A Senior Project Manager, fully loaded cost, no profit).  It is noted that this budget commitment is only 2018 even though the Met Office is committed to provide the Secretariat Function until 2020. 

Note that the action to move funds from the MFI bank account to the ING bank account already exists under this issue - https://github.com/OpenWIS/openwis-documentation/issues/344

b.	Review of need for additional finance

RESOLUTION – the SC resolved that no additional funding unanimously.

It is noted that 20,000 Euros will be received from ECMWF for 2017 and 2018 Associate Partner fees.

Post meeting note – The Board meeting clarified that ECMWF actually signed in January 2018.  Therefore, the amount of the invoice due from ECMWF needs to be checked.  The corresponding action is - (https://github.com/OpenWIS/openwis-documentation/issues/415)

c.	Preparation of budget for 2017/18

RESOLUTION – the SC resolved to spend 100,000 Euros on project related activities between now and the next annual meeting in 2019.

The SC anticipate the project to Integrate Commercial Search Engine to start in 2018.  The remaining two projects are anticipated to start in 2019.

RESOLUTION – the SC resolved to keep the reserve funds of £100k.

d.	Establishing bank account in Belgium for OpenWIS Association [Issue #343, Issue #344, Issue #346]

The SC agreed to increase the budget for four people to travel to ING Bank – 1600 Euros.

e.	Delegated financial authority for Managing Directors – proposed amendment to OpenWIS Financial Management Rules Supplement

RESOLUTION – Having reviewed the financial management rules supplement, the SC resolve that the existing limit of 10,000 Euros should not change.

SC-2018:#15 Recommendations to the Board to make and adopt, alter, supplement or repeal the Internal Rules

A new risk management supplement will be developed and recommended to the Board in due course. 
[Action-SC-2018-6 Create a risk management rules supplement]( https://github.com/OpenWIS/openwis-documentation/issues/410)

SC-2018:#16 Recommendations to the Board regarding the creation of ad-hoc committees or subsidiary bodies (including PMCs for new Projects)

The SC agreed the creation of four new PMCs.  Please refer to the discussion in the TC for details of Project Leads.

SC-2018:#17 Outstanding actions [refer to Annex 1]

Action-SC-2016-13: Seek legal advice on shelter from litigation [Issue #171]. The SC agreed and understood the legal position.
Action-SC-2017-36 Review the Community Manager role [Issue #207]  SC agreed that there is no requirement for a Community Manager
Action-SC-2017-82 Check the Yearbook entry; update if it exists [Issue #352, Issue #358]. The SC agreed that no further action should be taken.

SC-2018:#18 Next meeting

The SC agreed to have regular meetings, as they are useful for information sharing.  The aim is for quarterly meetings.

The SC agreed to schedule quarterly tele-conferences and cancel if there is nothing important to discuss.  
[Action-SC-2018-13 schedule SC meetings quarterly](https://github.com/OpenWIS/openwis-documentation/issues/418)

The SC agreed that the next SC being a tele-conference in early July 2018 to: discuss Project Charters for the four new projects; review the new OpenWIS Association slide deck from RGr; and review the new rule supplement for risk management.

[Action-SC-2018-14 topics for July 18 SC]( https://github.com/OpenWIS/openwis-documentation/issues/419)

Face to face meetings – The NWS is the next host.  

The SC agreed that SO and KS to check whether hosting is possible.  KS will report at the next SC (teleconference in July 2018).  The SC are aiming for March 2018.

[Action-SC-2018-15 next annual meeting](https://github.com/OpenWIS/openwis-documentation/issues/420)

SC-2018:#20 Summary of recommendations to the Board
a.	Pending from SC 2017-Dec [Issue #185]
•	[PR #340] Changes to describe amended governance arrangements. 

1.	Recommended to Board 
•	[PR #337] Added new internal to (almost) allow default 2-year tenure.

2.	Recommended to Board
 An update the ‘Stronger together’ presentation by end of June 2018 in order to discuss at the SC in early July 2018 is required.
[Action-SC-2018-16 update the 'stronger together' presentation]( https://github.com/OpenWIS/openwis-documentation/issues/421)

An agenda item on monthly TC meetings to call for papers or discussion topics on activities which may relate to topics relevant to WIS2 from TC attendees is required.
[Action-SC-2018-17 new standing TC agenda item re WIS2 info]( https://github.com/OpenWIS/openwis-documentation/issues/422)

SC-2018:#21 Closure of the meeting
The meeting closed at 18:14.

**Comment by rogers492** [comment]
Closed by #475
